examples/simple.py

```diff
@@ -13,15 +13,19 @@
 # function is used to actually generate the token
 @app.route('/login', methods=['POST'])
 def login():
-    username = request.json.get('username', None)
-    password = request.json.get('password', None)
-    if username != 'test' or password != 'test':
-        return jsonify({"msg": "Bad username or password"}), 401
-
-    # Identity can be any data that is json serializable
-    ret = {'access_token': create_access_token(identity=username)}
-    return jsonify(ret), 200
-
+    if request.is_json:
+        params = request.get_json()
+        if 'username' in params.keys() and 'password' in params.keys():
```

Comment:
Can you change this to not use `.keys()`? Doing `'username' in params'` is more efficient (at least with python2), as `.keys()` makes a copy of the keys into a list, which you then have to iterate through (O(n) instead of O(1))